release-notes/EPM/I-DOCSIS/EPM_6.1.7_I-DOCSIS.md

```diff
@@ -118,4 +118,4 @@ To resolve this, a number of changes were implemented to the Generic CM Collecto
 - Exception values have been added to several columns in the cable modem table.
 - Instead of the previous table cleanup method, the cable modem table is now cleaned up when the element starts up.
 
```

Comment:
There is also a change done on the Skyline EPM Platform and Skyline EPM Platform DOCSIS connectors :

Added exception values to several columns of the cable modem overview table.

Reply:
updated!